content/community.html

```diff
@@ -16,8 +16,8 @@ <h1>Community</h1>
     <p>
       <strong>User mailing lists:</strong>
       <ul>
-        <li><a href="https://groups.google.com/forum/#!forum/prometheus-announce">prometheus-announce</a> – low-traffic list for announcements like new releases.</li>
-        <li><a href="https://groups.google.com/forum/#!forum/prometheus-users">prometheus-users</a> – for discussions around Prometheus usage and community support. Announcements are <em>not</em> generally mirrored from <a href="https://groups.google.com/forum/#!forum/prometheus-announce">prometheus-announce</a>.</li>
+        <li><a href="https://groups.google.com/forum/#!forum/prometheus-announce">prometheus-announce</a> <a href="https://www.mail-archive.com/prometheus-announce@googlegroups.com/">(mirror)</a> – low-traffic list for announcements like new releases.</li>
```

Comment:
nit: Please move the parentheses from the link to regular text, so that it becomes

```
... (<a href="...">mirror</a>) ...
```